### ISSUE TITLE
Docs: Provide explicit examples for GitHub UI helpers

### ISSUE THREAD

**Issue by matijaSos** [wasp-lang/wasp#970]
[Right now](https://github.com/wasp-lang/wasp/blob/main/web/docs/language/features.md?plain=1#L1312), when talking about auth UI helpers for GitHub, we just refer readers to check out how Google works.

It's not wrong, but it also isn't the best dev experience - it's nicer to see an example of code you can immediately copy/paste. Also, I'm never sure how is "github" spelled, what is capitalized etc.

@shayneczyzewski @vincanger to your attention - lmk if this makes sense or if I missed something.

**Comment by shayneczyzewski** [comment]
Good point @matijaSos. The thinking there was just to avoid duplication, but it is nice to have copy-pasteable code.

@vincanger do we have any ideas on how we can provide some nice, full examples of things that will hopefully also be easy to maintain over time? I don't have a great idea myself :D so mainly just asking. But it would be cool if we had some type of simple web app perhaps that users could check boxes, and it would build out a .wasp file for them (and maybe accompanying js/ts) based on their selections. Too much though? 🤔

**Comment by sodic** [comment]
Commenting just to bump the issue. The links are outadated (and it's potentially solved).

**Comment by cprecioso** [comment]
Came to say the same thing as @sodic, this issue seems to be solved already.

@Martinsos wdyt?

**Comment by FranjoMindek** [comment]
This is solved. We no longer point to other Google on GitHub.
